### ISSUE TITLE
no asset_user title is displayed on RDP taskbar windows

### ISSUE THREAD

**Issue by PeretzEli** [OneIdentity/SCALUS#75]
the RDP minimized-window title only shows the IP of the destination server (when scalus is utilized)
when downloading and running the RDP file from SPP, the Asset_user title is displayed. (which is much preferred) 
attached a picture that shows the difference.
thanks in advance for any help on this
![scalusIssue](https://user-images.githubusercontent.com/109276331/178940557-659026a0-8091-4622-9f13-1c2c91fca5ed.png)

**Comment by Sqweeksyg** [comment]
Yes, It should be nice to have the DNS name instead of the ip of the asset.

**Comment by sgundlach3** [comment]
We are running into the same issue. Is there any update? @JaredCuster @PalwinderSinghQuest

**Comment by MattBeaudinQC** [comment]
Hello!

There is actually a enhancement request open (437659) with Safeguard for Privileged Sessions to add this functionality. I recommend that you open a support case and add your company's name to that request.

Thanks,

Matt
